scripts/docker/official/nginx-plus-with-nginx-agent/alpine/Dockerfile

```diff
@@ -0,0 +1,68 @@
+FROM alpine:3.17
```

Comment:
Any reason not to use official nginx/foss alpine image as a base here? It should be pretty straightforward and would make future maintenance easier.

Reply:
This Dockerfile is for NGINX Plus where there isn't an official image. We used the reference here to build an example Dockerfile that people can use if they are using NGINX Plus. https://docs.nginx.com/nginx/admin-guide/installing-nginx/installing-nginx-docker/#creating-nginx-plus-docker-image 

We do use the docker image for NGINX OSS alright here for this other Dockerfile we have https://github.com/nginx/agent/blob/fc8606e8ad47b83a830a77c067d52116d86f7bae/scripts/docker/official/nginx-oss-with-nginx-agent/alpine/Dockerfile

Reply:
I'm trying to say that the only difference between nginx/foss image and N+ image is nginx package itself; it's much easier to take the same base image (foss one) and just replace the package from nginx to nginx-plus. That way you won't need to keep and maintain separate Dockerfile. YMMV.

Reply:
The size of the resulting image would probably be larger if the package is replaced.

Reply:
Yes the size of the image would be bigger. Also there isn't mush difference in terms of number of lines in the Dockerfile with either approach. I would prefer to follow the dockerfile provided by nginx-plus to be consistent.